tests/resources/Harbor-Pages/Configuration.robot

```diff
@@ -278,7 +278,7 @@ Delete A Label
 ## Garbage Collection	
 Switch To Garbage Collection
     Sleep  1
-    Click Element  xpath=${configuration_xpath}
+    Click Element  xpath=//clr-vertical-nav-group-children/a[contains(.,'Garbage')]
```

Comment:
Do not use hard code for xpath locator, please use variable, and variable should be located in element file.

Reply:
Click Element  should be replace by anther keyword with retry function.